documentation/docs/25-build-and-deploy/50-adapter-static.md

```diff
@@ -70,7 +72,7 @@ The directory to write static assets (the contents of `static`, plus client-side
 
 ### fallback
 
-Specify a fallback page for [SPA mode](single-page-apps), e.g. `index.html` or `200.html` or `404.html`.
+To create a [single page app (SPA)](single-page-apps) you must specify the name of the fallback page to be generated by SvelteKit, which is used as the entry point for URLs that have not been prerendered. This is commonly `200.html`, but can vary depending on your deployment platform. You should avoid `index.html` where possible to avoid conflicting with a prerendered homepage. Note that this option has large negative performance and SEO impacts and is recommended only in very limited circumstances such as when being wrapped in a mobile app. See the [single page apps](single-page-apps) documentation for more details and alternatives.
```

Comment:
```suggestion
To create a [single page app (SPA)](single-page-apps) you must specify the name of the fallback page to be generated by SvelteKit, which is used as the entry point for URLs that have not been prerendered. This is commonly `200.html`, but can vary depending on your deployment platform. You should avoid `index.html` where possible to avoid conflicting with a prerendered homepage. 

> This option has large negative performance and SEO impacts and is recommended only in very limited circumstances such as when being wrapped in a mobile app. See the [single page apps](single-page-apps) documentation for more details and alternatives.
```